src/js/dynload.ts

```diff
@@ -157,74 +200,9 @@ export async function loadDynlibsFromPackage(
     pkg.file_name.split("-")[0]
   }.libs`;
 
-  // This prevents from reading large libraries multiple times. We may read it
-  // once in calculateGlobalLibs and again in loadDynlib.
-  // Memoized function is intentionally instanced per call to
-  // loadDynlibsFromPackage since we won't need the data again after we're done.
-  const readFileMemoized: ReadFileType = memoize(Module.FS.readFile);
```

Comment:
Really great to get rid of this @ryanking13. Thanks for writing it and also for removing it =)